tests/test_abort.js

```diff
@@ -21,8 +22,8 @@ test('req.abort() should cause "abort" and "error" to be emitted', t => {
     })
     .once('error', err => {
       // Should trigger last
-      t.equal(err.code, 'ECONNRESET')
-      t.ok(gotAbort, "didn't get abort event")
+      expect(err.code).to.equal('ECONNRESET')
+      expect(gotAbort).to.equal(true, "didn't get abort event")
```

Comment:
Do you think we need to keep these alternate message strings? I guess this one is worth something, though overall I'm not sure they are worth the effort to maintain. Also we could use sinon-chai with mocks and assert called, which probably would yield a good enough error message without having to craft one.

Reply:
I think this is a textbook case for a spy, sinon is my go-to. 
I would like to be generally cautious on using sinon for mocking internals. I like that we test out API surface.
Let me add it for this test and we can decide if we like it more with/without